src/main/java/uk/gov/hmcts/reform/civil/utils/HmcDataUtils.java

```diff
@@ -60,4 +64,48 @@ public static boolean hearingDataChanged(PartiesNotifiedResponse partiesNotified
             || partiesNotified.getServiceData() == null
             || hearingDataChanged(hearing, partiesNotified.getServiceData());
     }
+
+    private static int actualHours(int hearingDuration) {
```

Comment:
hey the method names are not clear with what they're doing. they look very similar and it's hard to understand the difference between the methods.

- Could we have java docs for examples of the strings created? or else something to make it clearer?